Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -147,8 +147,10 @@
     setInsertionPointColor(parameters.insertionPointColor);
     setHardwareKeyboardState(parameters.hardwareKeyboardState);
 #endif
-    WebCore::setAdditionalSupportedImageTypes(parameters.additionalSupportedImageTypes);
-    WebCore::setImageSourceAllowableTypes(WebCore::allowableImageTypes());
+    if (!WebProcess::singleton().isLockdownModeEnabled()) {
+        WebCore::setAdditionalSupportedImageTypes(parameters.additionalSupportedImageTypes);
```

Comment:
It looks like `WebCore::setAdditionalSupportedImageTypes` is also called from `-[WebView _preferencesChanged]`. Do we also need a Lockdown Mode check there?

Reply:
You right I restored the check for `isLockdownModeEnabled()` in `setAdditionalSupportedImageTypes()`. This should take care of not setting ant additional image types in lockdown mode.